parsl/usage_tracking/usage.py

```diff
@@ -117,21 +116,16 @@ def __init__(self, dfk, port=50077,
     def check_tracking_enabled(self):
         """Check if tracking is enabled.
 
-        Tracking will be enabled unless either of these is true:
+        Tracking will be enabled unless the following is true:
 
             1. dfk.config.usage_tracking is set to False
-            2. Environment variable PARSL_TRACKING is set to false (case insensitive)
 
         """
         track = True
 
         if not self.config.usage_tracking:
```

Comment:
this logic can be tidied up a lot! I think this function returns exactly the value of self.config.usage_tracking now? so the `if` statement and local variable are not needed at all?

Reply:
changed this to return `self.config.usage_tracking`